addons/account/models/account_invoice.py

```diff
@@ -319,6 +320,10 @@ def _compute_payments(self):
     outstanding_credits_debits_widget = fields.Text(compute='_get_outstanding_info_JSON')
     payments_widget = fields.Text(compute='_get_payment_info_JSON')
     has_outstanding = fields.Boolean(compute='_get_outstanding_info_JSON')
+    cash_rounding_id = fields.Many2one('account.cash.rounding', string='Cash Rounding',
+        states={'draft': [('readonly', False)]},
+        help = 'The smallest coinage of the currency used to pay by cash.')
+    allow_cash_rounding = fields.Boolean(string='Cash Rounding', related="company_id.allow_cash_rounding")
```

Comment:
would use a group to hide menuitems and fields. Users will be automatically added to that group when thicking option in settings